README.md

```diff
@@ -1,3 +1,98 @@
-# excubiae
+# Excubiae
 
-an on-chain flexible &amp; modular framework for implementing custom gatekeepers.
+Excubiae is a composable framework for building custom attribute-based access control policies on Ethereum.
+
+The framework provides a set of common, abstract, and flexible smart contracts to simplify the creation of reusable, attribute-based criteria. These contracts, called "gatekeepers," are solution-agnostic, enforcing checks against user-provided evidence and maintaining records of those who meet the criteria.
```

Comment:
seems like it's missing a word after attribute-based criteria? like attribute-based criteria policies maybe?

Reply:
Also thinking how this sentence could be restructured to not repeat too much of the previous one?